src/io.c

```diff
@@ -195,14 +195,47 @@ Char GET_NEXT_CHAR(void)
     }
     else
         STATE(In)++;
+
+    // handle line continuation, i.e., backslash followed by new line or CRLF;
+    // and also the end of a line in general
+    while (*STATE(In) == '\\' || *STATE(In) == 0) {
+        if (!*STATE(In))
+            GetLine();
+        else if (STATE(In)[1] == '\n') {
+            STATE(In) += 2;
+            /* print only a partial prompt from now on                             */
+            if ( !SyQuiet )
+              STATE(Prompt) = "> ";
+            else
+              STATE(Prompt) = "";
+        }
+        else if (STATE(In)[1] == '\r')
+            STATE(In) += (STATE(In)[2] == '\n') ? 3 : 2;
+        else
+            break;
+    }
+
+    return *STATE(In);
+}
+
+static Char GET_NEXT_CHAR2(void)
```

Comment:
Add a comment which explains that this internal function is like `GET_NEXT_CHAR`, except that it does not care about line coninuations. Also, come up with a better name, perhaps???

Alternatively, merge this code into `IGNORE_REST_OF_LINE`?